ORBIT/phases/design/SemiTaut_mooring_system_design.py

```diff
@@ -0,0 +1,167 @@
+__copyright__ = "Copyright 2020, National Renewable Energy Laboratory"
+__maintainer__ = "Jake Nunemaker"
+__email__ = "jake.nunemaker@nrel.gov & rebecca.fuchs@nrel.gov"
+
+from scipy.interpolate import interp1d
+import numpy as np
+
+from ORBIT.phases.design import DesignPhase
+
+
+class SemiTautMooringSystemDesign(DesignPhase):
+    """Mooring System and Anchor Design."""
+
+    expected_config = {
+        "site": {"depth": "float"},
+        "turbine": {"turbine_rating": "int | float"},
+        "plant": {"num_turbines": "int"},
+        "mooring_system_design": {
+            "num_lines": "int | float (optional, default: 4)",
+            "anchor_type": "str (optional, default: 'Drag Embedment')",
+            "mooring_line_cost_rate": "int | float (optional)",
+            "drag_embedment_fixed_length": "int | float (optional, default: .5km)",
+        },
+    }
+
+    output_config = {
+        "mooring_system": {
+            "num_lines": "int",
+            #"line_diam": "m, float", # this is not needed for mooring.py
+            "line_mass": "t", # you need this for mooring.py (mooring installation module)
+            "line_cost": "USD", # you can calculate this based on each rope&chain length & diameter.
+            "line_length": "m", # this can be calculated from rope length and chain length (which you get from an empirical eqn as function of depth)
+            "anchor_mass": "t", # you need this for mooring.py (mooring installation module) 
+            "anchor_type": "str", # keep, changed default to drag embedment.
+            "anchor_cost": "USD", # this can be calculated also as a function of (depth?) from the empirical data you have.
+        }
+    }
+
+    def __init__(self, config, **kwargs):
+        """
+        Creates an instance of MooringSystemDesign.
+
+        Parameters
+        ----------
+        config : dict
+        """
+
+        config = self.initialize_library(config, **kwargs)
+        self.config = self.validate_config(config)
+        self.num_turbines = self.config["plant"]["num_turbines"]
+
+        self._design = self.config.get("mooring_system_design", {})
+        self.num_lines = self._design.get("num_lines", 4)
+        self.anchor_type = self._design.get("anchor_type", "Drag Embedment")
+
+        self._outputs = {}
+
+    def run(self):
+        """
+        Main run function.
+        """
+
+        self.calculate_line_length_mass()
+        self.determine_mooring_line_cost()
+        self.calculate_anchor_mass_cost()
+
+        self._outputs["mooring_system"] = {**self.design_result}
+
+    def calculate_line_length_mass(self):
+        """
+        Returns the mooring line length and mass.
+        """
+
+        depth = self.config["site"]["depth"]
+
+        # Input hybrid mooring system design from Cooperman et al. (2022), https://www.nrel.gov/docs/fy22osti/82341.pdf 'Assessment of Offshore Wind Energy Leasing Areas for Humboldt and Moorow Bay Wind Energy Areas, California
```

Comment:
I agree. But since this file is going to removed eventually, I decided to leave this file alone. All the features and proper formats are done in `mooring_system_design.py`